packages/ui-components/src/components.d.ts

```diff
@@ -0,0 +1,4 @@
+/**
```

Comment:
@miguelpinto98 I added this placeholder file. This is to fix the CI failing issue due to importing a missing file. Please tell me what you think 👍